include/sapi/tss2_tcti.h

```diff
@@ -165,6 +165,23 @@ TSS2_TCTI_POLL_HANDLE *handles, size_t *num_handles);
 
 typedef TSS2_TCTI_CONTEXT_COMMON_V1 TSS2_TCTI_CONTEXT_COMMON_CURRENT;
 
+#define TCTI_INFO_SYMBOL "Tss2_Tcti_Info"
```

Comment:
I was wondering if this should be namespaced similar to the rest; i.e. TSS2_TCTI_INFO_SYMBOL since it is part of tss2_tcti.h?

Apologies for not finding it in the first run.

Reply:
Already merged, but will have PR for this change in a few moments.